packages/manager/apps/web/client/app/domain/webhosting/order/domain-webhosting-order.html

```diff
@@ -1,8 +1,12 @@
 <div data-ovh-alert="webhosting-order-alert"></div>
 
-<oui-back-button data-on-click="$ctrl.goBackToDashboard()">
-    <span data-translate="domain_webhosting_order_title"></span>
-</oui-back-button>
+<oui-header
+    data-heading="{{ 'domain_webhosting_order_title' | translate }}"
+    data-description="{{ 'domain_webhosting_order_description' | translate }}"
+>
+    <oui-back-button data-on-click="$ctrl.goBackToDashboard()">
+    </oui-back-button>
+</oui-header>
```

Comment:
`oui-back-button` inside `oui-header` element, it's not a mistake ?

Reply:
No I can put in.
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-header-webcomponent--with-back-button